### PR TITLE
Revert "CC177472, Adding missing breakline"

### PR DESCRIPTION
**Pull request by tdykstra**
Reverts dotnet/docs#16320

### CODE REVIEW

**Comment by mairaw** [comment]
I think the change made was good. Runtime and framework libraries are separate things, no?

**Comment by tdykstra** [comment]
That's what I thought at first, but read the bullet points again and you'll notice that they are all grammatically parallel -- "The [part name] which [does x, y, and z]", and the sentence starting ".NET Core framework libraries" doesn't fit the pattern.  ".NET Core framework libraries" expands on "type system ... basic services" in the first sentence of bullet 1.

**Comment by tdykstra** [comment]
Interesting to observe that this understanding matches the new repo arrangement as well -- the "runtime" repo includes the "libraries" as well as the runtime proper.

File: docs/core/about.md [hunk 1]
```diff
@@ -50,8 +50,7 @@ Multiple frameworks have been built on top of .NET Core:
...
 
-- The [.NET Core runtime](https://github.com/dotnet/runtime/tree/master/src/coreclr), which provides a type system, assembly loading, a garbage collector, native interop, and other basic services.
-- [.NET Core framework libraries](https://github.com/dotnet/runtime/tree/master/src/libraries) provide primitive data types, app composition types, and fundamental utilities.
+- The [.NET Core runtime](https://github.com/dotnet/runtime/tree/master/src/coreclr), which provides a type system, assembly loading, a garbage collector, native interop, and other basic services. [.NET Core framework libraries](https://github.com/dotnet/runtime/tree/master/src/libraries) provide primitive data types, app composition types, and fundamental utilities.
 - The [ASP.NET runtime](https://github.com/aspnet/home), which provides a framework for building modern cloud-based internet connected applications, such as web apps, IoT apps, and mobile backends.
 - The [.NET Core CLI tools](https://github.com/dotnet/cli) and language compilers ([Roslyn](https://github.com/dotnet/roslyn) and [F#](https://github.com/microsoft/visualfsharp)) that enable the .NET Core developer experience.
```
Comment (mairaw) [inline docs/core/about.md]:
```suggestion
- The [.NET Core CLI tools](https://github.com/dotnet/sdk) and language compilers ([Roslyn](https://github.com/dotnet/roslyn) and [F#](https://github.com/microsoft/visualfsharp)) that enable the .NET Core developer experience.
```

**Review by mairaw — APPROVED** [review]
I'll let you decide on what looks best. Not feeling strongly about one way or the other.

**Pull request merged**